docs/pages/validator-management/vc-configuration.md

```diff
@@ -88,11 +88,29 @@ Configure your validator client's fee recipient address by using the `--suggeste
 
 You may choose to use the `--strictFeeRecipientCheck` flag to enable a strict check of the fee recipient address with the one returned by the beacon node for added reassurance.
 
+### Configure your builder selection and/or builder boost factor
+
+If you are running a beacon node with connected builder relays, you may use these validator configurations to signal which block (builder vs. local execution) the beacon node should produce.
+
+With produceBlockV3 (enabled automatically after the Deneb hard fork), the `builder.boostFactor` is a percentage multiplier the block producing beacon node must apply to boost (>100) or dampen (<100) builder block value for selection against execution block. The multiplier is ignored if `--builder.selection` is set to anything other than `maxprofit`. Even though this is set on the validator client, the calculation is applied on the beacon node itself. For more information, see the [produceBlockV3 Beacon API](https://ethereum.github.io/beacon-APIs/#/ValidatorRequiredApi/produceBlockV3).
+
+With Lodestar's `--builder.selection` options, you can select:
+- `maxprofit`: Default setting for Lodestar set at `builder.boostFactor=100`. This default setting will always choose the more profitable block. Using this option, you may customize your `builder.boostFactor` to your preference. Examples of its usage are below.
+- `executiononly`: Validator will not trigger builder block production even if builder relays are configured on the beacon. This will always select the local execution block.
+- `builderalways`: An alias of `builder.boostFactor=18446744073709551615`, which will select the builder block, unless the builder block fails to produce. The builder block may fail to produce if it's not available, not timely or there is an indication of censorship via `shouldOverrideBuilder` from the execution payload response.
```

Comment:
Really minor but just noticed we might wanna add 2**64 - 1 here, else nobody would get what `18446744073709551615` means

maybe like this
```suggestion
- `builderalways`: An alias of `builder.boostFactor=18446744073709551615` (2**64 - 1), which will select the builder block, unless the builder block fails to produce. The builder block may fail to produce if it's not available, not timely or there is an indication of censorship via `shouldOverrideBuilder` from the execution payload response.
```